src/scripts.js

```diff
@@ -1,168 +1,167 @@
-      </div>`)
+  recipes.forEach(recipe => {
+    cardArea.insertAdjacentHTML('afterbegin', constructCard(recipe))
+  });
+  getFavorites();
+};
+
+const getFavorites = () => {
+  if (domUser.favoriteRecipes.length) {
+    domUser.favoriteRecipes.forEach(recipe => {
+      document.querySelector(`.favorite${recipe.id}`).classList.add('favorite-active')
     })
   }
 }
 
-function greetUser() {
-  const userName = document.querySelector('.user-name');
-  userName.innerHTML =
-  user.name.split(' ')[0] + ' ' + user.name.split(' ')[1][0];
-}
+const constructCard = (recipe, addedClass) => {
+  return `<div id='${recipe.id}'
+  class='card'>
+  <header id='${recipe.id}' class='card-header'>
+  <label for='add-button' class='hidden'>Click to add recipe</label>
+  <button id='${recipe.id}' aria-label='add-button' class='add-button card-button'>
+  <img id='${recipe.id}' class='add'
+  src='https://image.flaticon.com/icons/svg/32/32339.svg' alt='Add to
+  recipes to cook'></button>
+  <label for='favorite-button' class='hidden'>Click to favorite recipe
+  </label>
+  <button id='${recipe.id}' aria-label='favorite-button' class='favorite card-button ${addedClass}'>
+  </button></header>
+  <span id='${recipe.id}' class='recipe-name'>${recipe.name}</span>
+  <img id='${recipe.id}' tabindex='0' class='card-picture'
+  src='${recipe.image}' alt='Food from recipe'>
+  </div>`
+  //The over use of ID's makes me nervous but idk if it's a real issue
+};
 
-function favoriteCard(event) {
+const favoriteCard = (event) => {
   let specificRecipe = cookbook.recipes.find(recipe => {
     if (recipe.id  === Number(event.target.id)) {
       return recipe;
     }
   })
   if (!event.target.classList.contains('favorite-active')) {
-    event.target.classList.add('favorite-active');
-    favButton.innerHTML = 'View Favorites';
-    user.addToFavorites(specificRecipe);
+    event.target.classList.add('favorite-active'); //may be a use case for toggle
+    // showFavoritesButton.innerHTML = 'View Favorites'; seems not needed
+    domUser.addToFavorites(specificRecipe);
   } else if (event.target.classList.contains('favorite-active')) {
     event.target.classList.remove('favorite-active');
-    user.removeFromFavorites(specificRecipe)
+    domUser.removeFromFavorites(specificRecipe)
   }
-}
+};
 
-function cardButtonConditionals(event) {
+const addToToCook = (event) => {
+  let specificRecipe = cookbook.recipes.find(recipe => {
+    if (recipe.id  === Number(event.target.id)) {
+      return recipe;
+    }
+  });
+  if (!event.target.classList.contains('add-active')) {
+    event.target.classList.add('add-active'); //add-active isn't a class styling
+    domUser.addToRecipesToCook(specificRecipe);
+  } else if (event.target.classList.contains('add-active')) {
+    event.target.classList.remove('add-active');
+    domUser.haveCookedThisRecipe(specificRecipe) //may want two functions here one to remove and one to cook that meal
+  };
+};
+
```

Comment:
I wonder if we want a hidden check box (or similar) labeled "I've cooked this recipe!". It would only appear if the recipe was added to the toCook list. If checked, that would trigger removal of the recipe from the list and removal of used ingredients from the user pantry.

Reply:
^ I like this idea.

Reply:
This is a great idea! Added this to our wire frame to see a few ways we could interact with that.